### ISSUE TITLE
Warning from MessageSender.onSendComplete

### ISSUE THREAD

**Issue by adoelger** [Azure/azure-event-hubs#169]
When using sendSync I get sporadically the following warning:

Jun 15, 2016 7:34:34 AM com.microsoft.azure.servicebus.MessageSender onSendComplete
WARNING: path[gpsehdev], linkName[d83d6a_624a7127620a480fb27efb667bfcf7cd_G13], delivery[edee92bff90c4b25a3b904a1900e72ee] - mismatch

What does it mean? Is the event written into the event hub or not? I'm using the latest reales (0.7.1).

**Comment by SreeramGarlapati** [comment]
@adoelger - This happened due to an accounting issue with DeliveryTag's in MessageSender. We fixed this (in the 0.7.4 Realease Candidate) - please give it a shot and let us know:

```
<repository> 
<id>azure eventhubs</id> 
<url>https://oss.sonatype.org/content/repositories/commicrosoftazure-1333/</url> 
</repository>
<dependency>
<groupId>com.microsoft.azure</groupId>
<artifactId>azure-eventhubs</artifactId>
<version>0.7.4</version>
</dependency>
```

**Comment by SreeramGarlapati** [comment]
closing the issue - due to lack of followup response.
Please comment - if you do see any problem...
